pricewars_merchant.py

```diff
@@ -125,8 +126,14 @@ def update_offers(self) -> None:
 
     def restock(self):
         order = self.producer.order(self.settings['restock limit'])
```

Comment:
We have setting keys like `'restock limit'` and `'primeShipping'`. We should decide for either camel case or underscores (or even spaces, but we should unify all keys).